plugins/qCompass/ccMouseCircle.cpp

```diff
@@ -0,0 +1,155 @@
+#include "ccMouseCircle.h"
+
+#include <math.h>
+
+ccMouseCircle::ccMouseCircle(ccGLWindow* owner, QString name) : cc2DViewportObject(name.isEmpty() ? "label" : name)
+{
+	setVisible(true);
+	setEnabled(false);
+
+	//setup unit circle
+	float heading;
```

Comment:
Move declaration into loop since you aren't using it outside loop.
(`float heading = n ...`)